package-lock.json

```diff
@@ -1,8 +1,9468 @@
 {
   "name": "@workablehr/orka",
-  "version": "2.5.4",
-  "lockfileVersion": 1,
+  "version": "2.5.5",
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
+    "": {
```

Comment:
@kkyriakidis what is this? How was it included? Can you please revert with a new pr?

Reply:
CC @manoskouvarakis is going to update the package-lock again and remove this. @kkyriakidis, @echetzakis  any clues how it was included in the first place?

Reply:
@kkyriakidis @echetzakis This is the new version of package-lock if installed with npm 7 or later:
https://nitayneeman.com/posts/catching-up-with-package-lockfile-changes-in-npm-v7/